Formula/dgraph.rb

```diff
@@ -3,6 +3,9 @@ class Dgraph < Formula
   homepage "https://dgraph.io"
   url "https://github.com/dgraph-io/dgraph/archive/v20.03.3.tar.gz"
   sha256 "1e2145d8921b980882ef4475c82337e80c15fa3012a012424dadc9f636bffaa1"
+  # Source code in this repository is variously licensed under the Apache Public License 2.0 (APL)
+  # and the Dgraph Community License (DCL). A copy of each license can be found in the licenses directory.
+  license "Apache-2.0"
```

Comment:
Is it worth waiting for https://github.com/Homebrew/brew/pull/7953?

Reply:
we can certainly take an iterative approach.

When it is dual licensed, I often chose the less permissive one.

Reply:
another interesting thing with the license problem is the dependencies, like `ffmpeg`

Reply:
Just to make sure our thought process is documented on GitHub:

Dual license issues will be addressed in the future.

For now, the less permissive license was selected as the one license for the formula.